_docs/setup/using-initializer.md

```diff
@@ -0,0 +1,333 @@
+```
+```bash
+x-request-id=a641eff7-eb82-4a4f-b67b-53cd3a03c399
+```
+```bash
+kubectl logs ${CLIENT} proxy | grep a641eff7-eb82-4a4f-b67b-53cd3a03c399
+```
+```bash
+[2017-05-01T22:08:39.310Z] "GET / HTTP/1.1" 200 - 0 398 3 3 "-" "curl/7.47.0" "a641eff7-eb82-4a4f-b67b-53cd3a03c399" "service-two" "10.4.180.7:8080"
+```
+
+## Understanding what happened
+
+Here's what happened after the workload was submitted to Kubernetes:
+
+1) kubernetes adds `sidecar.initializer.istio.io` to the list of
+pending initializers in the workload.
+
+2) istio-initializer controller observes a new uninitialized workload was created. 
+It finds its configured name `sidecar.initializer.istio.io`
+as the first in the list of pending initializers.
+
+3) istio-initializer checks to see if it was responsible for
+initializing the namespace of the workload. No further work is done
+and the initializer ignores the workload if the initializer is
+configured for the namespace. By default the initializer is
+responsible for all namespaces (see #extended-configuration)
+
+4) istio-initializer removes itself from the list of pending
+initializers. Kubernetes will not finish creating workloads if the
+list of pending initializers is non-empty. A misconfigured initializer
+means a broken cluster.
+
+5) istio-initializer checks the default injection policy for the mesh
+_and_ any possible per-workload overrides to determine whether the
+sidecar proxy should be injected.
+
+6) istio-initializer injects the sidecar template into the workload
+and submits it back to kubernetes via PATCH.
+
+7) kubernetes finishes creating the workload as normal and the
+workload includes the injected sidecar proxy.
+
+## [Additional configuration options](#extended-configuration)
+
+The istio-initializer has a global default policy for injection as
+well as per-workload overrides. The global policy is configured by the
+`istio-inject` ConfigMap (see example below). The initializer pod must
+be restarted to adopt new configuration changes.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: istio-inject
+  namespace: istio-system
+data:
+  config: |-
+    policy: "enabled"
+    namespaces: [""] # everything, aka v1.NamepsaceAll, aka cluster-wide
+    initializerName: "sidecar.initializer.istio.io"
+    params:
+      initImage: docker.io/istio/proxy_init:0.2.4
+      proxyImage: docker.io/istio/proxy:0.2.4
+      verbosity: 2
+      version: 0.2.4
+      meshConfigMapName: istio
+      imagePullPolicy: IfNotPresent
+```
+
+### _policy_
+
+`off` - Disable the initializer from modifying resources. The pending
+'status.sidecar.istio.io initializer' initializer is still removed to
+avoid blocking creation of resources.
+
+`disabled` - The initializer will not inject the sidecar into
+resources by default for the namespace(s) being watched. Resources can
+enable injection using the "sidecar.istio.io/inject" annotation with
```

Comment:
use code instead of quotations - `sidecar.istio.io/inject` (same for instance in line 278)